labs/shell.md

```diff
@@ -79,58 +81,77 @@ In addition, we can follow our process execution, failures, times and some other
 
 ## Part 2: Self Contained Application
 
-This section was run from the System shell:
+In this section we are going to deploy an app to a Spark cluster.
 
-    cd sampleapp && ../sbt/bin/sbt package
+*Exercises*
 
-    $SPARK_HOME/bin/spark-submit \
-    --master spark://127.0.0.1:7077 \
-    --total-executor-cores 4 \
-    target/scala-2.11/sample-app_2.11-1.0.jar "../data/README.md" \
-    --class "SampleApp"
+* Navigate to the sampleApp folder and create the dist package.
+		
+		cd sampleapp && ../sbt/bin/sbt package
+
+* Submits the dist package to the master spark node, defining 4 nodes and specifying the main class name of your app.
```

Comment:
I would write:

`Submits the dist package to the master spark node, requesting 4 CPU cores along all the available executors and specifying the main class name of your app.`